.flowconfig

```diff
@@ -1,6 +1,6 @@
-[version]
-0.125.1
-
+# [version]
+# 0.126.1
+#
```

Comment:
I'll reinstate this once I've finished the upgrade process (though, I don't really see the point in keeping this and wonder if we should drop it so dependabot can auto update us).